packages/graphql/tests/integration/issues/2267.int.test.ts

```diff
@@ -71,9 +71,9 @@ describe("https://github.com/neo4j/graphql/issues/2267", () => {
         await session.run(`
         CREATE(:${Place} {displayName: "786 aa"})
         CREATE(:${Place} {displayName: "8 à Huita"})
-        CREATE(:${Place} {displayName: "9ème Sauvagea"})<-[:ACTIVITY]-(:${Story} {name: "A story"})
-        CREATE(:${Place} {displayName: "A One Shopa"})
         CREATE(:${Place} {displayName: "zaza"})
+        CREATE(:${Place} {displayName: "A One Shopa"})
+        CREATE(:${Place} {displayName: "9ème Sauvagea"})<-[:ACTIVITY]-(:${Story} {name: "A story"})
```

Comment:
Why this change?

This test is quite delicate, as it is testing ordering, by changing the order the elements were created you may be changing the outcome of the test if the limit is lacking here

Could this be the reason this test is now passing despite reverting the changes?

Reply:
It may worth include the 2267 TCK tests to highlight the initial reason for the changes

Reply:
The test was always passing - I reordered things here to see if that was the reason the test didn't start failing following the change. We were creating the nodes in alphabetical order, and then querying them ordering them by this same order - so I am just removing this uncertainty.

Reply:
Fair enough :+1: